docker_test.go

```diff
@@ -417,8 +418,71 @@ func run(m *testing.M) (int, error) {
 	}); err != nil {
 		return 0, fmt.Errorf("Could not connect to postgres %q: %w", DB_DSN, err)
 	}
-	fmt.Println("DB_DSN:", DB_DSN)
-	// ----------
+	log.Println("DB_DSN:", DB_DSN)
+	// Set  timescale DB 
+	// pulls an image, creates a container based on it and runs it
+	database = "rs_postgres"
+	timescaleRes, err := pool.RunWithOptions(&dockertest.RunOptions{
+		Repository:   "timescale/timescaledb",
+		Tag:          "latest-pg13",
+		ExposedPorts: []string{"5432"},
+		Env: []string{
+			"POSTGRES_USER=postgres",
+			"POSTGRES_DB=" + database,
+			"POSTGRES_PASSWORD=password",
+		},
+	})
+	if err != nil {
+		return 0, fmt.Errorf("Could not start resource transformer: %w", err)
+	}
+	defer func() {
+		if err := pool.Purge(timescaleRes); err != nil {
+			log.Printf("Could not purge resource: %s \n", err)
+		}
+	}()
+	timescaleDB_DSN := fmt.Sprintf("postgresql://postgres:password@host.docker.internal:%s/%s?sslmode=disable", timescaleRes.GetPort("5432/tcp"), database)
+	fmt.Println("timesacaleDB_DSN",timescaleDB_DSN)
+	timescaleDB_DSN_1 := fmt.Sprintf("postgresql://postgres:password@localhost:%s/%s?sslmode=disable", timescaleRes.GetPort("5432/tcp"), database)
+	if err := pool.Retry(func() error {
```

Comment:
Suggestion, for better readability:
```suggestion
	timescaleDB_DSN_Internal := fmt.Sprintf("postgresql://postgres:password@host.docker.internal:%s/%s?sslmode=disable", timescaleRes.GetPort("5432/tcp"), database)
	fmt.Println("timesacaleDB_DSN",timescaleDB_DSN)
	timescaleDB_DSN_viaHost := fmt.Sprintf("postgresql://postgres:password@localhost:%s/%s?sslmode=disable", timescaleRes.GetPort("5432/tcp"), database)
	if err := pool.Retry(func() error {
```

Reply:
Also, consider making `timescaleDB_DSN_viaHost` a local if it is used only for readiness